config.py

```diff
@@ -71,6 +72,20 @@ def read_configs(directory=CURRENT_DIRECTORY):
         )
         configs["broker_ip"] = DEFAULT_BROKER_IP
 
+    try:
+        # Transform env var string "1280,740" into int tuple (1280, 740)
+        configs["viewport_size"] = tuple(
+            map(int, os.getenv("VIEWPORT_SIZE").split(","))
+        )
+    except (AttributeError, ValueError):
+        # AttributeError for if getenv returns None,
+        # ValueError for failed int cast
+        warn(
+            "VIEWPORT_SIZE is not set in .env, "
+            + f"defaulting to {DEFAULT_VIEWPORT_SIZE}"
+        )
```

Comment:
It seems like both errors give the same error message which can be confusing. Might want to use a more general error message like "Error when loading VIEWPORT_SIZE" or split the Error handling to have one message for each.